pycaching/cache.py

```diff
@@ -613,6 +616,20 @@ def description(self, description):
         description = str(description).strip()
         self._description = description
 
+    @property
+    @lazy_loaded
+    def description_html(self):
+        """The cache long description in raw HTML.
+
+        :type: :class:`str`
+        """
+        return self._description
+
+    @description.setter
```

Comment:
This should be the setter for the `description_html` property. At the moment, you have two setters for `description`.

Reply:
Just fixed!